config/routes.rb

```diff
@@ -9,13 +9,18 @@
       resources :learning_resources
       resources :reviews, except: [:index]
     end
-    resources :groups
+    resources :groups # TODO: this line seems like is not required
     resources :documents
     resources :groups do
       resources :attendances, except: [:edit, :update, :destroy]
     end
     resources :students_learning_resources, only: [:new, :create]
   end
 
-  resources :students, except: [:index, :create, :destroy]
+  resources :students, except: [:index, :create, :destroy] do
+    scope module: :students do
+      resources :learning_reasources, only: [:index]
+      resources :attendances, only: [:index]
+    end
```

Comment:
I'm wondering if we should have routes for `reviews` here, let's talk about it tomorrow

Reply:
As we talked, this is already fixed!